test/test_wizard.py

```diff
@@ -104,17 +112,20 @@ def test_wizard_get_href_project(self):
     def test_wizard_run_checks_project(self):
         project = self.get_project()
         project['info']['task_presenter'] = ''
+        # default value guidelines <p><br></p>
+        project['info']['task_guidelines'] = '<p><br></p>'
```

Comment:
Similar to comment above, regarding the default value or an empty string. Thanks!